http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java

```diff
@@ -89,6 +90,8 @@ public class HttpServerConfig
     private int adminMaxThreads = 200;
 
     private boolean showStackTrace = true;
+    private boolean showErrorInTitle = true;
```

Comment:
Do we need these new configs? Having a single config seems fine -- show stack traces or not. Especially considering that we want to replace the error handler in Trino, I don't see a need for extra configuration.